pkg/apis/monitoring/v1/types.go

```diff
@@ -916,7 +916,7 @@ type NativeHistogramConfig struct {
 	// It requires Prometheus >= v2.45.0.
 	//
 	// +optional
-	ScrapeClassicHistograms *bool `json:"scrapeClassicHistograms,omitempty"`
+	AlwaysScrapeClassicHistograms *bool `json:"alwaysScrapeClassicHistograms,omitempty"`
```

Comment:
We can't change the name of the field in the CRD, it would be a breaking change. I suggest to keep the current name but the generated configuration needs to take the Prometheus version into account.

Reply:
I thanks for the review, changed as requested, could you pls check again. Thanks